app/lib/server/startup/settings.js

```diff
@@ -2752,4 +2752,19 @@ settings.addGroup('Rate Limiter', function() {
 	});
 });
 
+settings.addGroup('Alexa', function() {
+	this.add('Register_Alexa_Enable_Server', false, {
+		type: 'boolean',
+		public: true,
+	});
+	this.add('Register_Alexa_Enable_Server_Proxy_URL', '', {
+		type: 'string',
+		public: true,
```

Comment:
This setting is not used on client side so it should not be public

Reply:
This still requires a fix

Reply:
Its is actually used here => https://github.com/RocketChat/Rocket.Chat/pull/15084/files#diff-f7e8b26d04245ac2c60d80cfbaeede03R81

Reply:
@PrajvalRaval you are using `Register_Alexa_Enable_Server` there and not the `Register_Alexa_Enable_Server_Proxy_URL` setting right?

Reply:
I am using `Register_Alexa_Enable_Server_Proxy_URL` setting here => https://github.com/RocketChat/Rocket.Chat/pull/15084/files#diff-fe30fa511449edb34519d15783b3930cR23

Reply:
I believe I can remove 'public = true' for `Register_Alexa_Enable_Server_Proxy_URL` since its been used on server side. Let me check.